build.sbt

```diff
@@ -47,6 +47,7 @@ def commonScalacOptions(scalaVersion: String) = {
     "-language:implicitConversions",
     "-language:experimental.macros",
     "-unchecked",
+    "-Ypartial-unification",
```

Comment:
What do we need this for? Let's avoid adding compiler flags without a really solid reason why.

Reply:
Same as above.